test/e2e/operator_test.go

```diff
@@ -87,6 +88,84 @@ func TestDefaultClusterIngressExists(t *testing.T) {
 	}
 }
 
+func TestClusterProxyProtocol(t *testing.T) {
```

Comment:
Is this providing much more value than the unit test in this case?

I'm trying to think of how we could make it more black-box either here or in Origin... like, is the feature really "source IP is preserved on supported cloud environments" (PROXY being an implementation detail)?

Reply:
I don't want to block on it right now, just food for thought

Reply:
That makes sense - have to rebase anyway so will fix the test then. Probably in the next few days (too many tests + needle pricks today to be in a sane frame of mind to do this right now).

Reply:
Aah, we need a service to return back the header info. Plus there's already an extended test for router headers (in the origin repo) that checks the remote ip  -- it is disabled currently - but will work to fix on it there.
Leaving this test here for now (we can at least verify that the proxy protocol works once we have all the pieces in place). Thx